vcenter/vcenter.py

```diff
@@ -16,17 +18,14 @@ class VCenter():
 
     def __init__(self):
         self._connected = False
+        self._connection_cookie = None
         self.content = None
         self.__logger = logging.getLogger(__name__)
         self.vm_folders = None
 
     def __check_connection(self):
-        try:
-            objView = self.content.viewManager.CreateContainerView(self.content.rootFolder,
-                                                                   [vim.Datastore],
-                                                                   True)
-            objView.Destroy()
-        except Exception:
+        result = self.__get_objects_list_from_container(self.content.rootFolder, vim.Datastore)
+        if not result:
```

Comment:
as `__get_objects_list_from_container` always returns empty list in the worst case, this is really ellegant solution 👍